README.md

```diff
@@ -23,9 +23,53 @@ Or download a prebuilt binary from [releases](./releases).
 ```bash
 protoc 
   --typescript-http_out [OUTPUT DIR] \
+  --typescript-http_opt use_enum_numbers=true,use_multi_line_comment=true
   [.proto files ...]
 ```
 
+#### Support options
+
+```ts
+// UseProtoNames controls the casing of generated field names.
+// If set to true, fields will use proto names (typically snake_case).
+// If omitted or set to false, fields will use JSON names (typically camelCase).
+use_proto_names: bool
+
+// UseEnumNumbers emits enum values as numbers.
+use_enum_numbers: bool
+
+// The method names of service methods naming case.
+// Only work when `UseEnumNumbers=true`
+// opt:
+// camelcase: convert name to lower camel case like `camelCase`
+// pascalcase: convert name to pascalcase like `PascalCase`
+// default is pascalcase
+enum_field_naming: string
+
+// Generate comments as multiline comments.
+// multiline comments: /** ... */
+// single line comments: // ...
+use_multi_line_comment: bool
+
+// force add `undefined` to message field.
+// default true
+force_message_field_undefinable: bool
+
+// If set to true, body will be JSON.stringify before send
+// default true
+use_body_stringify: bool
```

Comment:
While I'm all for having this option, I understand it as you are also using that as a way to pass options to the request handler? 

I would rather see a new argument to the request handler - something like `handlerOptions: T` - than spreading it into the request body.

Reply:
noop, at this monent,  `JSON.stringify` is a mandatory behavior.

https://github.com/einride/protoc-gen-typescript-http/blob/d11ee374e8c0bc1b61365a437dc0e97eea143020/internal/plugin/servicegen.go#L167-L170

add this option is remove it, and the default value is `true` to avoid break change.

if someone like to use `JSON.stringify`, this option can be enabled, or archive it in `requestHandler`.